pkg/utils/gwlog/actions.go

```diff
@@ -0,0 +1,4 @@
+package gwlog
+
+const ReconcileStart = "[ACTION_RECONCILE_START]"
+const ReconcileEnd = "[ACTION_RECONCILE_END]"
```

Comment:
Just thinking this one through. For "easy" querying, should these just be `"RECONCILE_START"` and `"RECONCILE_END"`? We don't have those strings anywhere else in the code, so should be OK and then wouldn't potentially need escaping for the square brackets.

Reply:
adding _MARKER as a suffix selfishly as our internal logging tool is case insensitive. just in case someone wants to log reconcile_start etc.